spokestack/tts/clients/spokestack.py

```diff
@@ -10,7 +10,7 @@
 import requests
 
 _MODES = {
-    "ssml": "synthesizeSSML",
+    "ssml": "synthesizeSsml",
```

Comment:
Interestingly enough, this appears to work both ways, but this version is more consistent with the other platforms and the formatting in the GraphQL schema.